### ISSUE TITLE
Compose key as optional feature (settings)

### ISSUE THREAD

**Issue by Validbit** [Julow/Unexpected-Keyboard#630]
While this feature is helpful for many cases, it can be often mistyped (as it's on navigation key).

I suggest 

- [x] creation a setting entry, like:

Include Compose key
_Adds key for quick creation of complex symbols_

- [ ] setting it off (by default)

Users could **in their own pace** get (introduced) to it and people with more complex/foreign accents would likely find it in the settings as it's short enough imo.

**Comment by Spike-from-NH** [comment]
Yes, it's startling if mistyped. Keys go dim! a helpful feature, but only if you want it.

I've made it a swipe (of Rubout) and never hit it by accident. This may be another solution.

**Comment by Julow** [comment]
The compose key can be disabled in the "Add keys to the keyboard" option, where it's enabled by default.

I agree that the position is not ideal but I'm not willing to hide the compose key by default. For many, the keyboard do not allow yet to access all the needed characters and not having the compose key ready is an extra barrier.
I'm open to suggestion on where to move this key. I'm also considering removing the dpad key in favor of up-down and left-right sliders (perhaps as an option in the next-next release) depending on how impactful the recent improvements to the slider were, that would also have an impact to this issue.

**Comment by Julow** [comment]
I think this issue can be closed. Thanks!
